### ISSUE TITLE
Function of lookup button

### ISSUE THREAD

**Issue by arogl** [faulander/P4S#5]
What is the function of the lookup button, it currently does nothing, unless related to my other issues

![image](https://user-images.githubusercontent.com/1115472/72419043-c1f06c00-37c7-11ea-85e6-7375b7950322.png)

**Comment by faulander** [comment]
yes, that's true. Sonarr can only add shows, which have already a THETVDB ID. If TVMaze doesn't provide that, i need to implement a search function - which i will do. It's just not finished.
